docs/modules/ROOT/pages/tutorials/getting-started.adoc

```diff
@@ -1,483 +1,186 @@
-----
-curl -H "$LIEUTENANT_AUTH" "http://${LIEUTENANT_URL}/tenants"
-kubectl -n lieutenant get tenant
-kubectl -n lieutenant get gitrepo
-----
+The command above returns something like the following:
 
-. Register a *Lieutenant Cluster* via the API
-+
-[source,shell]
-----
-CLUSTER_ID=$(curl -s -H "$LIEUTENANT_AUTH" -H "Content-Type: application/json" -X POST \
-  --data "{
-            \"tenant\": \"${TENANT_ID}\",
-            \"displayName\": \"My first Project Syn cluster\",
-            \"facts\": {
-              \"cloud\": \"local\",
-              \"distribution\": \"k3s\",
-              \"region\": \"local\"
-            },
-            \"gitRepo\": {
-              \"url\": \"ssh://git@${GITLAB_ENDPOINT}/${GITLAB_USERNAME}/cluster-gitops1.git\"
-          }}" \
-  "http://${LIEUTENANT_URL}/clusters" | jq -r ".id")
-echo $CLUSTER_ID
-echo https://${GITLAB_ENDPOINT}/${GITLAB_USERNAME}/cluster-gitops1
-----
-+
-[TIP]
+[source]
 --
-If everything went well, the Lieutenant Operator created a new git repository under https://${GITLAB_ENDPOINT}/${GITLAB_USERNAME}/cluster-gitops1 which will be used to store the generated catalog of deployment files.
+NAME             DISPLAY NAME         AGE
+t-aged-pond-24   Project Syn Tenant   13s
 --
 
-. Retrieve the registered Clusters via API and directly on the cluster
-+
-[source,shell]
-----
-curl -H "$LIEUTENANT_AUTH" "http://${LIEUTENANT_URL}/clusters"
-kubectl -n lieutenant get cluster
-kubectl -n lieutenant get gitrepo
-----
-
-== Kickstart Commodore
-
-Commodore is the configuration generation tool. It will be configured to generate configuration for your Lieutenant cluster `$CLUSTER_ID` generated above. With all the information available in Lieutenant, Commodore is able to figure out what to actually compile for the cluster in question and where to Git push the compiled catalog to.
+In the context of Project Syn, a "tenant" is an entity to whom clusters are assigned. Tenants could be customers, departments, teams, or any entity responsible for owning a cluster.
 
-Before continuing with this section, make sure that everything went well with the installation and configuration of Lieutenant as Commodore relies on having a working instance of it.
+Project Syn stores cluster-specific configuration values in that tenant's own configuration Git repository. You should now have a new project in your GitLab account called "Project Syn Tenant."
 
-=== Run Commodore
+== Steward
 
-The easiest way of executing Commodore is by using the container image provided by Project Syn: https://hub.docker.com/r/projectsyn/commodore[docker.io/projectsyn/commodore]. We run the image directly in the local `k3s` or `docker-desktop` instance so that there is no need for having another container runtime installed.
+Steward is the in-cluster agent of Project Syn. It runs in each cluster associated with a tenant, automatically reacting to changes in the configuration stored in the corresponding Git repositories.
```

Comment:
That is only part of it. The heavy lifting is actually done by Argo CD. Steward only does the initial bootstrap of Argo CD and takes care of facts reporting to Lieutenant.

Reply:
That's explained below. I'd rather not go into much depth at this level.